src/basic_recipes/barplot.jl

```diff
@@ -208,7 +209,7 @@ function Makie.plot!(p::BarPlot)
 
         if stack === automatic
             if fillto === automatic
-                fillto = 0.0
+                fillto = offset
             end
         elseif eltype(stack) <: Integer
             fillto === automatic || @warn "Ignore keyword fillto when keyword stack is provided"
```

Comment:
```suggestion
            if !iszero(offset)
                @warn "Ignore keyword offset when keyword stack is provided"
                offset = 0.0
            end
```

I think in the future we should support `offset` in combination with `stack` (just move the whole stacked bar up or down), but the current implementation doesn't work, so I think it's best to just throw a warning and remove the offset in that case. We can add the feature in a future PR.

Reply:
Actually, I wonder if just changing line 219 to `y, fillto = to, from .+ offset` is enough to get `offset` and `stack` to play nice together, maybe that's the best solution.